nautobot_ssot/contrib/adapter.py

```diff
@@ -168,6 +168,10 @@ def load(self):
             raise ValueError("'top_level' needs to be set on the class.")
 
         for model_name in self.top_level:
+            if hasattr(self, f"load_{model_name}"):
```

Comment:
we should document this behavior for developers... but why not just removing the model from the top_level if you don't want to load it?

Reply:
@chadell The point is we still want to load the model, but be able to customize the functionality when loading on a per-model basis. 

I'll get started on the docs.

Reply:
@chadell looking into it, it doesn't appear there's any docs yet for using contrib. Because of that, I think it would be best to save the docs portion for a separate issue/PR.

Reply:
@Renrut5 The docs for contrib are under Modeling in the User Guide section.